tests/test_release_transpiler_argument_types.py

```diff
@@ -0,0 +1,564 @@
+import argparse
+import sys
+from contextlib import contextmanager
+from os.path import abspath, dirname
+
+import pytest
+
+from komodo.release_transpiler import main as release_transpiler_main
+
+RELEASE_TRANSPILER_FILE_PATH = abspath(
```

Comment:
this constant is probably unused?

Reply:
I was unsure about this one, I feel like main could be misunderstood and someone might assume main is the main function of the test file; not the release_transpiler_main.
Should I remove the constant and just use main instead?

Reply:
It is the constant `RELEASE_TRANSPILER_FILE_PATH` that I guess is unused.

Reply:
Okay, I misunderstood. Is this something pylint should have caught?